core/base/CMakeLists.txt

```diff
@@ -6,24 +6,211 @@ ROOT_ADD_TEST_SUBDIRECTORY(test)
 
 include_directories(${CMAKE_CURRENT_SOURCE_DIR}/res ${CMAKE_CURRENT_SOURCE_DIR}/../foundation/res)
 
-ROOT_GLOB_HEADERS(Base_dict_headers ${CMAKE_CURRENT_SOURCE_DIR}/inc/LinkDef?.h
-                                    ${CMAKE_CURRENT_SOURCE_DIR}/inc/T*.h
-                                    ${CMAKE_CURRENT_SOURCE_DIR}/inc/Rtypes.h
-                                    ${CMAKE_CURRENT_SOURCE_DIR}/inc/Gtypes.h
-                                    ${CMAKE_CURRENT_SOURCE_DIR}/inc/GuiTypes.h
-                                    ${CMAKE_CURRENT_SOURCE_DIR}/inc/MessageTypes.h
-                                    ${CMAKE_CURRENT_SOURCE_DIR}/inc/KeySymbols.h
-                                    ${CMAKE_CURRENT_SOURCE_DIR}/inc/Buttons.h
-				    ${CMAKE_CURRENT_SOURCE_DIR}/inc/ROOT/TSequentialExecutor.hxx)
+set(Base_dict_headers LinkDef?.h
```

Comment:
The '?' is a form of globbing.  Shouldn't we expand it too.

Reply:
thank you @pcanal!